src/toil/batchSystems/abstractBatchSystem.py

```diff
@@ -47,6 +48,12 @@ def __init__(self, config, maxCores, maxMemory, maxDisk):
         """
         :type dict[str,str]
         """
+        self.workerCleanupInfo = {
```

Comment:
I'd prefer a namedtuple here. Any reason you changed it back to dict?

Also take a look at the dict() built-in with keyword arguments. It makes for more consise dicts literals if their keys resemble Python identifiers. But again, I'd prefer a namedtuple.